src/ripple/app/ledger/LedgerHistory.cpp

```diff
@@ -384,7 +391,7 @@ void LedgerHistory::handleMismatch (
         log_one (validLedger, (*v)->key(), "built", j_);
 }
 
-void LedgerHistory::builtLedger (Ledger::ref ledger)
+void LedgerHistory::builtLedger (Ledger::ref ledger, Json::Value&& consensus)
 {
```

Comment:
That will still create an extra shallow object, but I guess the reduced complexity is worth it.